lib/ddtrace/contrib/rails/patcher.rb

```diff
@@ -40,7 +40,7 @@ def patched?
           end
 
           def compatible?
-            return if ENV['DISABLE_DATADOG_RAILS']
+            return if ENV['DISABLE_DATADOG_RAILS']&.match(/^(true|t|yes|y|1|disable|disabled)?$/i)
```

Comment:
Isn't it confusing if DISABLE_DATADOG_RAILS=disable or disabled ?

Reply:
Oh, yes, you are right. `true` should not be `disabled`.

I guess, ENABLE_DATADOG_RAILS would have been less confusing than double negation...

How about this:
`return if ENV['DISABLE_DATADOG_RAILS'] && ENV['DISABLE_DATADOG_RAILS'] != '0'`

Reply:
`return if ENV['DISABLE_DATADOG_RAILS'] && ENV['DISABLE_DATADOG_RAILS'] != '0'` looks great 👍